package.json

```diff
@@ -1,7 +1,28 @@
 {
+  "name": "tumblr.js",
+  "version": "1.0.0",
+  "description": "Official JavaScript client for the Tumblr API",
+  "main": "./lib/tumblr",
+  "scripts": {
+    "test": "gulp"
+  },
+  "homepage": "https://www.github.com/tumblr/tumblr.js",
+  "repository": {
+    "type": "git",
+    "url": "https://www.github.com/tumblr/tumblr.js"
```

Comment:
I think this is supposed [to end in `.git`](https://docs.npmjs.com/files/package.json#repository). The recommended link on Github is `https://github.com/tumblr/tumblr.js.git`